src/views/collection-view/server.ts

```diff
@@ -31,7 +31,7 @@ export interface CollectionPagePath {
  * which is needed for other areas of the app to function.
  */
 export async function getCollectionPageProps(
-  product: ProductData,
+  product: LearnProductData,
```

Comment:
Having `LearnProductData` as the type here ensures that the slug being passed to the API call is the right one